stdlib/SparseArrays/test/sparsevector.jl

```diff
@@ -75,7 +75,11 @@ end
 @testset "other constructors" begin
     # construct empty sparse vector
 
-    @test exact_equal(spzeros(Float64, 8), SparseVector(8, Int[], Float64[]))
+    for dims in (8, (8,))
+        @test exact_equal(spzeros(dims), SparseVector(8, Int[], Float64[]))
+        @test exact_equal(spzeros(Float64, dims), SparseVector(8, Int[], Float64[]))
+        @test exact_equal(spzeros(Float64, Int, dims), SparseVector(8, Int[], Float64[]))
```

Comment:
```suggestion
        @test exact_equal(spzeros(Float64, Int16, dims), SparseVector(8, Int16[], Float64[]))
```
Let's have some non-standard thing here to make sure we see the effect of the extra argument, and then merge.